src/app/post-thread-page/post-thread/post-thread.component.ts

```diff
@@ -27,7 +28,9 @@ export class PostThreadComponent {
     public globalVars: GlobalVarsService,
     private backendApi: BackendApiService,
     private changeRef: ChangeDetectorRef,
-    private toastr: ToastrService
+    private toastr: ToastrService,
+    private titleService: Title,
+    private metaService:Meta,
```

Comment:
No longer need `Meta` & `MetaDefinition` to be imported in all the files, as you are just using `Title`.